sandbox/user_guide.md

```diff
@@ -257,7 +257,52 @@ As shown above, in the field labeled "Resource types requested," type "Coverage.
 
 If you’d like to use the command line or implement this API call in code, look in the `Curl` section (shown in the image above) for the request you just made. Not far below that, you can see the response: an `HTTP 202 Accepted` giving a link in the content-location header for status information on your Coverage job.
 
-### 5. Getting your data
+### 5. Using the `_since` parameter
+The `_since` parameter lets you filter your bulk data requests by the date when it was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data. The API will return data updated between your `_since` input date and the present.
+
+Using the `_since` parameter comprises two steps:
+
+1. Input a date in the correct format.
+2. Start the job to acquire data from an endpoint.
+
+#### b. Input a date in the correct format.
+
+First, click “Try it Out” in the Swagger section for `_since`. Then, enter your desired date into the dialog box labeled "`_since` (Optional)". Dates and times submitted in `_since` must adhere to a specific format for the server to understand. That format is the FHIR _dateTime_ format (`YYYY-MM-DDThh:mm:ss+zz:zz`). Notice that, if you need to include a time, a timezone must also be specified (`+zz:zz`).
+
+To ensure that there will be no gaps in claims data that you receive using the `_since` parameter, it may be helpful to use the date of your most recent bulk data request for subsequent data pulls using the `_since` parameter. You may retrieve this date by viewing the [_transactionTime_](https://hl7.org/Fhir/uv/bulkdata/export/index.html#response---complete-status) from your last bulk data request.
+
+The example below demonstrates how to convert a date/time combination into the FHIR format.
+
+**Date and Time Example**
+
+<img class="ug-img" src="/assets/img/since_Screenshot_1.png" alt="" />
```

Comment:
Are these the screenshots that are in the other PR?  I think they might have been committed to the wrong branch; they need to be in this branch 😄